helm/coder/values.yaml

```diff
@@ -265,6 +265,9 @@ coder:
     # coder.service.httpsNodePort -- Enabled if coder.service.type is set to NodePort.
     # If not set, Kubernetes will allocate a port from the default range, 30000-32767.
     httpsNodePort: ""
+    # coder.service.prometheusNodePort -- Enabled if coder.service.type is set to NodePort.
+    # If not set, Kubernetes will allocate a port from the default range, 30000-32767.
+    prometheusNodePort: ""
```

Comment:
This needs to mention that the prometheus port is only exposed in the service if CODER_PROMETHEUS_ENABLE=true in the env section.

Reply:
updated and added an example to the Prometheus documentation